frontend/src/layout/navigation/ProjectNotice.tsx

```diff
@@ -108,6 +108,15 @@ export function ProjectNotice(): JSX.Element | null {
                 children: 'Log out',
             },
         },
+        internet_connection_issue: {
+            message: 'PostHog is having trouble connecting to the server. Please check your connection.',
```

Comment:
This sounds at first read like _PostHog_ has a problem. I'd rephrase like:
```suggestion
            message: 'You're offline. Please check your connection.',
```